src/PrestaShopBundle/Resources/config/services/form/form_type.yml

```diff
@@ -334,6 +334,16 @@ services:
         tags:
             - { name: form.type }
 
+    form.type.order_preferences.general:
+        class: 'PrestaShopBundle\Form\Admin\Configure\ShopParameters\OrderPreferences\GeneralType'
+        parent: 'form.type.translatable.aware'
+        public: true
+        arguments:
+            - '@prestashop.adapter.data_provider.currency'
+            - '@=service("prestashop.adapter.data_provider.cms").getCmsChoices()'
```

Comment:
don't we need to pass the language id to the function here?

Reply:
the cms data provider internally uses CMS::listCms($idLang = null, $idBlock = false, $active = true),
it takes the default language if no argument is passed. In this case the default language should be used so there's no need to pass it as an argument
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/CMS.php#L166